tests/test_core/test_alignment.py

```diff
@@ -2546,17 +2551,17 @@ def test_deepcopy(self):
             self.assertEqual(len(new_seq.data), 10)
             self.assertTrue(new_seq.data.is_annotated())
             self.assertEqual(len(new_seq.data.annotations), 1)
-            # tests the case when just_span argument if False
-            new_seq = aln.AlignedSeqs[name].deepcopy(just_span=False)
+            # tests the case when sliced argument if False
+            new_seq = aln.AlignedSeqs[name].deepcopy(sliced=False)
             self.assertEqual(len(new_seq.data), len(aln.AlignedSeqs[name].data))
             self.assertTrue(new_seq.data.is_annotated())
             self.assertEqual(len(new_seq.data.annotations), 1)
         for name in ["NineBande", "DogFaced"]:
             new_seq = aln.AlignedSeqs[name].deepcopy()
             self.assertEqual(len(new_seq.data), 10)
             self.assertFalse(new_seq.data.is_annotated())
-            # tests the case when just_span argument if False
-            new_seq = aln.AlignedSeqs[name].deepcopy(just_span=False)
+            # tests the case when sliced argument if False
+            new_seq = aln.AlignedSeqs[name].deepcopy(sliced=False)
             self.assertEqual(len(new_seq.data), len(aln.AlignedSeqs[name].data))
             self.assertTrue(new_seq.data.is_annotated())
             self.assertEqual(len(new_seq.data.annotations), 1)
```

Comment:
after this last loop, check that `aln.named_seqs["Human"].deepcopy(sliced=False)` for 2 annotations.